src/tests/multihost/alltests/test_sssctl_analyzer.py

```diff
@@ -14,6 +14,15 @@
 from constants import ds_instance_name
 
 
+def execute_cmd_client(multihost, *commands):
```

Comment:
The "commands" handling is confusing, as it looks like multiple commands can be executed.
Please do use a variable raiseonerr with a default value instead of this.

Reply:
yes, rather than running the function again and again for every single command, here we can use single function call to run  multiple command in client like bellow.

        execute_cmd_client(multihost, "mkdir /tmp/sssd; "
                                      "cp -vf /var/log/sssd/* /tmp/sssd; "
                                      "chown user5000 /tmp/sssd/; "
                                      "chown user5000 /tmp/sssd/*")


Some times we need to put raiseonerr = False , for this feature to work we have to keep this function as it is , this gives us flexibility

Reply:
This would do the same job:
```
def execute_cmd_client(multihost, command, raiseonerr=True):
    return multihost.client[0].run_command(command, raiseonerr=raiseonerr)
```
If you are running multiple commands separated by ";" then You do not need a new cool function for that.
If you wanted to run multiple commands properly then the passing them in list and returning the results in a list or dictionary is the right way.

```
def execute_cmd_client(multihost, commands, raiseonerr=True):
    results = []
    for command in commands:
        results.append(multihost.client[0].run_command(command, raiseonerr=raiseonerr))
    return results
```
Note that the commands will have each proper result that can be checked and we will not have a silent/invisible failure like when running all of them separated by ";"


This way You can at least detect if it failed somewhere without it being silently ignored:
```
execute_cmd_client(multihost, "mkdir /tmp/sssd && "
                                  "cp -vf /var/log/sssd/* /tmp/sssd && "
                                  "chown user5000 /tmp/sssd/ &&"
                                  "chown user5000 /tmp/sssd/*")
```

This will do the same and do it better without introducing new functions:
```
multihost.client[0].run_command(
    "mkdir /tmp/sssd && "
     "cp -vf /var/log/sssd/* /tmp/sssd && "
     "chown user5000 /tmp/sssd/ &&"
     "chown user5000 /tmp/sssd/*",
    raiseonerr=True)
``` 
It will also support passing an input like single password.